packages/mds-agency/handlers/create-event.ts

```diff
@@ -138,20 +138,28 @@ export const createEventHandler = async (
     await refreshDeviceCache(device, event)
 
     const { telemetry } = event
+    let recorded_telemetry: Recorded<Telemetry> | undefined = undefined
     if (telemetry) {
```

Comment:
Come to think about it, there is *always* telemetry on events in 1.0! I think that a lot of this conditional code is completely unnecessary now.

Reply:
👍